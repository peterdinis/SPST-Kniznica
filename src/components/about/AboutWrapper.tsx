import Header from "../shared/Header";
import styles from "../../styles/Component.module.css";
import Paper from "@mui/material/Paper";

const AboutWrapper: React.FC = () => {
  return (
    <>
      <Header name="O Stránke" />
      <Paper elevation={3}>
        <h1 className={styles.ownerName}>Správca je ...</h1>
        <p className={styles.helperText}>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Id sequi et
          nesciunt sed excepturi quaerat, incidunt a architecto molestiae! Saepe
          iusto temporibus eaque voluptate officiis facere veniam amet
          praesentium id.
        </p>
      </Paper>
    </>
  );
};

export default AboutWrapper;
