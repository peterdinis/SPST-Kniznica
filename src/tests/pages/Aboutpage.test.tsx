import { render, screen } from '@testing-library/react'
import AboutWrapper from '../../components/about/AboutWrapper'
import AboutPage from "../../pages/about"

describe("Aboutpage Test", () => {

    // fix bug with not find components with prefix @ in jest.config
    it.skip("Render aboutpage", () => {
        const {container} = render(<AboutPage />);
        expect(container).toMatchSnapshot();
    })

    it("Render aboutwrapper", async () => {
        const {container} = render(<AboutWrapper />);
        expect(container).toMatchSnapshot();
    })


    it("Render text in paper component", async () => {
        return;
    })

    it("Test failed because requested text does not exists in aboutWrapper", async () => {
        return;
    })
})