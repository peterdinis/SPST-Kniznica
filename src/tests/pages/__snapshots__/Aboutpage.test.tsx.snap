// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aboutpage Test Render aboutpage 1`] = `
<div>
  <h1
    class="header"
  >
    O Stránke
  </h1>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation3 css-3su884-MuiPaper-root"
  >
    <h1
      class="ownerName"
    >
      Správca je ...
    </h1>
    <p
      class="helperText"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Id sequi et nesciunt sed excepturi quaerat, incidunt a architecto molestiae! Saepe iusto temporibus eaque voluptate officiis facere veniam amet praesentium id.
    </p>
    <div
      class="mt-4 text-center"
    >
      <a
        class="text-xl font-bold"
        href="/rooms"
      >
        Otázky pre adminov
      </a>
    </div>
  </div>
</div>
`;

exports[`Aboutpage Test Render aboutwrapper 1`] = `
<div>
  <h1
    class="header"
  >
    O Stránke
  </h1>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation3 css-3su884-MuiPaper-root"
  >
    <h1
      class="ownerName"
    >
      Správca je ...
    </h1>
    <p
      class="helperText"
    >
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Id sequi et nesciunt sed excepturi quaerat, incidunt a architecto molestiae! Saepe iusto temporibus eaque voluptate officiis facere veniam amet praesentium id.
    </p>
    <div
      class="mt-4 text-center"
    >
      <a
        class="text-xl font-bold"
        href="/rooms"
      >
        Otázky pre adminov
      </a>
    </div>
  </div>
</div>
`;
